### DISCUSSION TITLE
How do I know that `check_fares` is working?

### DISCUSSION THREAD

**Discussion by wootwoot1234** [jdholtz/auto-southwest-check-in#114] [Q&A] [answered]
I have enabled `check_fares: true` but I don't see anything in the logs that shows that it's checking the prices.  Will it show something every 24 hours saying it checked the prices?

**Comment by jdholtz** [comment, marked as answer]
By default, the script doesn’t output anything unless a lower flight fare is actually found. However, you can see the debug output of the fare checking process by running the script with the `--verbose` flag. With this output, you’ll be able to see the fare difference it finds for every flight.

**Comment by wootwoot1234** [comment]
That worked great, thanks!  I LOVE this feature!  

It would be nice to see some basic information in the logs like the check-in feature.

For example, on startup:
`Fare check enabled`

and when the fares are checked: 
`Fare check found a price change from 'City' to 'City' for First Last of +$XX`

and if you have a companion:
`Fare check is not supported with companion passes. Skipping fare check for 'City' to 'City' for First Last`

**Reply by jdholtz** [reply]
That might be a good idea to add different levels of verbosity. The reason I didn’t include that in the regular output is because it would run every day, so eventually the output would be filled with just those messages and you wouldn’t be able to see the scheduling and check-in messages.

**Reply by wootwoot1234** [reply]
That makes sense, it would be cool to have the option to view that data for people just setting it up.  

Some logs showing that fare check is enabled on initialization would be helpful

**Reply by jdholtz** [reply]
As long as you have `check_fares` equal to true in your config file, it will be enabled. Also, in debug mode, there is a message at the top saying where it is enabled or not.

**Comment by wootwoot1234** [comment]
It would be helpful to know which flights are being checked.  For example.  This is the log message but it doesn't tell me which one of my flights is checked:

`[fare_checker]: Flight price change found for 0 USD`

it would be nice if it included the city and/or date and/or confirmation code

**Reply by jdholtz** [reply]
The reason I didn’t do that is because I want people to be able to share their logs and therefore don’t want to log any sensitive information. I haven’t seen a reason for wanting to know a flight’s fare difference if it is higher than the one currently booked which is why I haven’t included it
